src/test/java/com/neo4j/docker/TestConfSettings.java

```diff
@@ -135,4 +136,53 @@ void testReadTheConfFile() throws Exception
         //Kill the container
         container.stop();
     }
+
```

Comment:
We have a test `testAdminConfOverride()` which checks that configs from environment variables override default settings. 
Could you add a test that checks configs from environment variables override configs from the neo4j.conf file?

It'd be pretty similar to the test `testConfigsAreNotOverridenByDockerentrypoint()` only you start the container with an environment variable for the same config as you set in the test conf file. Verify that the value used is the same as the environment variable and NOT the default and NOT the value from the conf.